### ISSUE TITLE
Mark everything untestd as EXPERIMENTAL_FEATURE

### ISSUE THREAD

**Issue by RudolfWeeber** [espressomd/espresso#2990]

**Comment by RudolfWeeber** [comment]
See #2071 for a list and links to coverage reporting

**Comment by RudolfWeeber** [comment]
Not everything untested is a compile time feature.

To still block the use of some stuff, we'd need a decorator

```
exp=False

def raise_exp(self,*args,**kwargs):
  raise Exception("nees exp: "+str(type(self)))

def dec(obj):
  if not exp:
    obj.__init__ = raise_exp
  return obj

@dec
class A:
  pass

A()
```

**Comment by jngrad** [comment]
Fixed by #3131
